libavcodec/h264_slice.c

```diff
@@ -1295,6 +1295,43 @@ static int h264_export_frame_props(H264Context *h)
         h->avctx->properties |= FF_CODEC_PROPERTY_CLOSED_CAPTIONS;
     }
 
+    if (h->sei.picture_timing.fulltc_received) {
```

Comment:
Need to set this to zero again, if we don't get an SEI on the next frame this will trigger.